biochemist-python/chapters/Non-Linear Regression Part 2.ipynb

```diff
@@ -0,0 +1,794 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# To generate a smoother fitted curve\n",
+    "smooth_x = np.linspace(np.min(xdata), np.max(xdata), 1000) # new x value range\n",
+    "plt.plot(xdata, ydata, 'o', label='data') # plot of the experimental data\n",
+    "plt.plot(smooth_x, MM(smooth_x, Vmax, Km), 'b', label = 'fit') # plot of the fit data\n",
```

Comment:
Move function call out of plotting. Combining statements like this usually confuses beginners.

Reply:
I created a new variable called smooth_y:

smooth_y = MM(smooth_x, Vmax, Km) # y values for the 1000 points in smooth_x

Then I used smooth_y for the y data in plt.plot.

plt.plot(smooth_x, smooth_y, 'b', label = 'fit') # plot of the fit data